translations/harbour-matkakortti-zh_CN.ts

```diff
@@ -0,0 +1,167 @@
+        <translation>该卡非 HSL 旅行卡</translation>
+    </message>
+    <message id="matkakortti-info-reading">
+        <source>Reading the card</source>
+        <extracomment>Info label</extracomment>
+        <translation>读取卡片</translation>
+    </message>
+    <message id="matkakortti-info-ready">
+        <source>Ready</source>
+        <extracomment>Info label</extracomment>
+        <translation>就绪</translation>
+    </message>
+    <message id="matkakortti-info-disabled">
+        <source>NFC is disabled</source>
+        <extracomment>Info label</extracomment>
+        <translation>已禁用 NFC</translation>
+    </message>
+    <message id="matkakortti-info-touch_hint">
+        <source>Place the phone on the card</source>
+        <extracomment>Hint label</extracomment>
+        <translation>将手机放置于卡片之上</translation>
+    </message>
+    <message id="matkakortti-card-header">
+        <source>Travel card</source>
+        <extracomment>Page title</extracomment>
+        <translation>旅行卡</translation>
+    </message>
+    <message id="matkakortti-time_unit_abbr-min">
+        <source>min</source>
+        <extracomment>Time unit (abbreviated)</extracomment>
+        <translation>分钟</translation>
+    </message>
+    <message id="matkakortti-time_unit_abbr-hour">
+        <source>h</source>
+        <extracomment>Time unit (abbreviated)</extracomment>
+        <translation>小时</translation>
+    </message>
+    <message id="matkakortti-time_unit_abbr-24hour">
+        <source>d</source>
+        <extracomment>Time unit (abbreviated)</extracomment>
+        <translation>天</translation>
+    </message>
+    <message id="matkakortti-time_unit_abbr-day">
+        <source>d</source>
+        <extracomment>Time unit (abbreviated)</extracomment>
+        <translation>天</translation>
+    </message>
+    <message id="matkakortti-switcher-details">
+        <source>Details</source>
+        <extracomment>Switcher label</extracomment>
+        <translation>详情</translation>
+    </message>
+    <message id="matkakortti-switcher-history">
+        <source>History</source>
+        <extracomment>Switcher label</extracomment>
+        <translation>历史</translation>
+    </message>
+    <message id="matkakortti-details-section-season_tickets">
+        <source>Season tickets</source>
+        <extracomment>Section header</extracomment>
+        <translation>季卡</translation>
+    </message>
+    <message id="matkakortti-details-zone">
+        <source>Zone:</source>
+        <extracomment>Label</extracomment>
+        <translation>区域：</translation>
+    </message>
+    <message id="matkakortti-details-section-card_value">
+        <source>Card value</source>
+        <extracomment>Section header</extracomment>
+        <translation>卡片金额</translation>
+    </message>
+    <message id="matkakortti-details-loaded_value-label">
+        <source>Last loaded value:</source>
+        <extracomment>Label</extracomment>
+        <translation>最近读取金额：</translation>
```

Comment:
Are you sure that you translated it correctly? This is the amount that you most recently added to (put on) your travel card. Google translates this as "Recently read amount:"